src/components/timelapse/TimelapseBar.vue

```diff
@@ -3,13 +3,17 @@
     <v-slider
       v-model="sliderVal"
       :tick-labels="ticksLabels"
-      :max="12"
+      :max="maxVal"
       step="1"
       ticks="always"
       tick-size="3"
       color='teal'
       always-dirty
-    ></v-slider>
+      :thumb-size="60"
+      thumb-label
+    >
+      <span slot="thumb-label" slot-scope="{value}">{{times[sliderVal]}}</span>
```

Comment:
https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots-with-the-slot-scope-Attribute